src/components/Toastify.tsx

```diff
@@ -104,7 +104,9 @@ export const enqueuePromiseToast = (
               href={getMultichainScanLink(
                 chainId,
                 data?.transactionHash ?? "",
-                "tx",
+                chainId === ChainId.EVMOS || chainId === ChainId.EVMOS_TESTNET
```

Comment:
is `txs` vs `tx` an evmos idiom?

Reply:
It is on mintscan https://www.mintscan.io/evmos/txs/11F47D7FD2AD6E43C1FFE7AB6A70D4987012B2D1F170D3B486AA92DF8EC89E44